docs/configuration.md

```diff
@@ -58,21 +62,34 @@ The `cluster-lock.json` has the following schema:
   "distributed_validators": [                               // Length equal to num_validators (n*32ETH staked).
     {
       "distributed_public_key":  "0x123..abfc",             // DV root pubkey
-      "public_shares": [ "0x123..abfc", "0x123..abfc"],     // The public share of each operator (length of num_operators)
-      "fee_recipient": "0x123..abfc"                        // Fee recipient address of this validator. Defaults to definition fee_recipient if empty.
+      "public_shares": ["0x123..abfc", "0x123..abfc"],      // The public share of each operator (length of num_operators)
+      "deposit_data": {...},                                // Deposit data to activate this validator
+      "builder_registration": {...}                         // Pre-generated signed builder registration for the validator
     }
   ],
   "lock_hash": "0xabcdef...abcedef",                        // Hash of the cluster definition and distributed validators. Uniquely identifies a cluster lock.
-  "signature_aggregate": "0xabcdef...abcedef"               // BLS aggregate signature of the lock hash signed by all the key shares of all the distributed validators. Proves that the key shares exist and attested to being part of this cluster.
+  "signature_aggregate": "0xabcdef...abcedef",               // BLS aggregate signature of the lock hash signed by all the key shares of all the distributed validators. Proves that the key shares exist and attested to being part of this cluster.
+  "node_signatures": ["0xabcdef...abcedef"]                 // Signatures of the lock hash for each operator defined in the cluster definition
 }
 ```
 
-`charon run` just requires a `cluster-lock.json` file to configure the cluster.
+`charon run` just requires a `cluster-lock.json` file to configure the cluster. See the [cluster.DepositData](../cluster/deposit.go) and [cluster.BuilderRegistration](../cluster/registration.go) Go structs for more details
+on how the an individual `distributed_validator` looks like.
 
 ### Cluster Config Change Log
 
 The following is the historical change log of the cluster config:
-- `v1.4.0` **default**:
+- `v1.7.0` **default**:
+  - Added the `builder_registration` structure to `distributed_validators` list in cluster lock.
+  - This enables distributed validators to submit pre-generated validator registrations that enables MEV-blocks.
+  - Added the `node_signatures` list to cluster lock which contains signatures of the lock hash signed by each individual node.
+- `v1.6.0`:
+  - Add `deposit_data` structure to `distributed_validators` list in cluster lock.
+  - This allows a creator to activate the validators as the deposit data for each validator is now included in the cluster lock.
```

Comment:
This allows anyone with access to the lock file to activate the validators, specifically, the creator.